forgerock-core/src/test/java/org/forgerock/android/auth/RootDetectorTest.kt

```diff
@@ -111,6 +112,8 @@ class RootDetectorTest {
 
     @Test
     fun testBusyBoxProgramFileDetector() {
+        // ubuntu-latest runner actually has BusyBox installed, so skip the test on Linux
+        assumeTrue(!System.getProperty("os.name").equals("Linux", ignoreCase = true))
```

Comment:
This unit test is failing on `ubuntu-latest` runner because BusyBox is actually installed on this runner. Previously (on `macos-latest`) it was not failing...